nimble/host/syscfg.yml

```diff
@@ -304,6 +308,7 @@ syscfg.defs:
         description: >
             Number of CoC channels allocated to EATT
         value: 0
+        restrictions: BLE_GATT_NOTIFY_MULTIPLE
```

Comment:
hmm where this requirement comes from?

Reply:
This setting eanbles EATT, and this restriction comes from ICS 1.4 GATT Features, Multiple Variable Length Notifications, status C.10:
```
Excluded IF SUM ICS 31/17 “Core v4.2” OR SUM ICS 31/18 “Core v4.2+HS” OR SUM ICS 31/19
“Core v5.0” OR SUM ICS 31/20 “Core v5.1”, otherwise Mandatory IF GATT 2/3a “Enhanced ATT
bearer over LE” OR GATT 2/3b “Enhanced ATT bearer over BR/EDR”, otherwise Optional.
```
so `Mandatory IF GATT 2/3a “Enhanced ATT bearer over LE` applies. So I think it's correct: to enable EATT restriction of enabled Multiple Notifications muist be fulfilled. Multiple Notifications can be also enabled without EATT, as optional